README.md

```diff
@@ -260,14 +301,18 @@ Code and documentation will be reviewed by Aaron R. Williams and possibly additi
     * Is the code DRY (don't repeat yourself)? If code is repeated more than once, recommend that the writer turn the repeated code into a function or macro.
     * Is there a place where a variable is rebuilt or changed later on?
     * Are values transcribed by hand?
+4. Public Release
+*Is the code clearly commented for public release (e.g., no use of abbreviations or acronyms that are Urban Institute specific)
+*Is the code free from any licenses, PII, or proprietary information. 
 
 ## How to Prepare for a Code Review
 
-* Data access should be abundantly clear. Scripts should download the data or instructions for the necessary files located on Box should be included. 
-* State if special computation was used (i.e. the Stata server or SAS server). 
-* If scripts use many variable names, make sure to include a codebook so reviewers can follow along.
-* Clearly comment each step with a focus on why the step is needed. 
-* Use clear object and file names. 
+* Data access should be abundantly clear. Scripts should download the data or instructions for the necessary files located on Box should be included (see Starting or raw data practices above). 
+* State if special computation was used when running the program (i.e. the Stata server or SAS server, anything off of a standard desktop). 
+* If scripts use many variable names, make sure to include a codebook so reviewers can follow along. Code books should include variable names along with definitions of the variable. For categorical variables, please include a list of the possible categories for the given variable.
+* Clearly comment each step in the program with a focus on why the step is needed. Make sure to explain the logic behind the step so that a reviewer can judge whether the code effectively accomplishes this.
+* Use clear object (names of data frames or macro variables) and file names. For file names see the file naming convention above. 
+*All scripts should run all the way through without errors. This should be the case regardless of the user/computer. 
 
 ## Code Reviews in GitHub
```

Comment:
This is where you'll want to add that all substantive correspondence about the coding takes place in GitHub issues.

Also, the creating final files needs to be updated. Or removed because you lead with the final files.

Reply:
We are using the term "final files" twice, we should distinguish. Here final files refers to the files that will end up on the data catalogue with all the metrics combined. For now I'm leaving this as final files and will refer to the earlier case as "final metric file"

Reply:
Good point! You're right. Yeah, getting a distinction between those two would be helpful. Thanks!